crowdin.yaml

```diff
@@ -36,9 +36,8 @@ files:
         nl: nl
         no: no
         pl: pl
-        pt: pt
-        pt-PT: pt
         pt-BR: pt-BR
+        pt-PT: pt
```

Comment:
Not sure this needs to be applied, I see french (`fr`, `fr-CA`) configured as this was and working just fine

Reply:
Ok, I'll remove this commit, and force push (I don't like the "Revert of a revert" concept in our git history 😅)